angular/angular.json

```diff
@@ -30,7 +30,7 @@
               }
             ],
             "styles": [
-              "../html/_spark.scss",
+              "../storybook-utilities/storybook-theming/scss/_spark-angular-manifest.scss",
```

Comment:
We weren't including the spark-angular scss file, I fixed that by creating a manifest that imports both spark and spark angular.